test/specs/apexLspPerf.perf.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { step } from 'mocha-steps';
+import { TextEditor } from 'wdio-vscode-service';
+import { TestSetup } from '../testSetup';
+import * as utilities from '../utilities';
+import { EnvironmentSettings } from '../environmentSettings';
+
+describe('Apex LSP Perf', async () => {
+  let testSetup: TestSetup;
+
+  step('Ensure Apex Language Server Starts Successfully', async () => {
+    utilities.log(`ApexLsp - Set up the testing environment`);
+    testSetup = new TestSetup(
+      'ApexLsp',
+      false,
+      EnvironmentSettings.getInstance().sfdxProjectHome
+    );
+
+    await testSetup.setUpTestingEnvironment();
+
+    const workbench = await browser.getWorkbench();
+
+    await utilities.runCommandFromCommandPrompt(
+      workbench,
+      'Developer: Show Running Extensions',
+      1
+    );
+
+    // Verify Apex Language Server extension is present and running
+    const extensionNameDivs = await $$('div.name');
```

Comment:
Lines 35-42: The latest version of the `develop` branch now has `/utilities/extensionUtils.ts` and has `findExtensionInRunningExtensionsList()`